instagram/src/App.js

```diff
@@ -0,0 +1,56 @@
+
+import React, { Component } from "react";
+import PostPage from "./components/PostContainer/PostPage";
+import dummyData from "./dummy-data";
+import Auth from './components/Auth/Auth';
+import "./App.css";
+ class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      comments: [],
+      likes: [],
+      filter: "",
+    };
+  }
+   componentDidMount() {
+    this.setState({
+      data: dummyData,
+      comments: dummyData.comments,
+      likes: dummyData.likes
+    });
+  }
+   handleInputChange = event => {
+    this.setState({
+      filter: event.target.value
+    });
+  };
+   filterItems = event => {
+    event.preventDefault();
+    if (this.state.filter === "") {
```

Comment:
it would have been better if you had two variables in state one is holding the data and other holds what to display.